modules/integration_azure-firewall/conf/01-snat.yaml

```
@@ -3,7 +3,7 @@ module: Azure firewall
 name: "SNAT Port Utilization"
 
 transformation: true
-aggregation: ".mean(by=['protocol'])"
+aggregation: ".mean(by=['protocol', 'azure_resource_name'])"
```

We commonly use this format

```suggestion
aggregation: ".mean(by=['protocol', 'azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
```

---

Thanks, it's fixed